recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -69,4 +69,4 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package_multi"] = "boost"
         self.cpp_info.filenames["cmake_find_package"] = "ut"
         self.cpp_info.filenames["cmake_find_package_multi"] = "ut"
-        self.cpp_info.components["ut"].includedirs = []
+        self.cpp_info.components["ut"]
```

Comment:
The line looked awkward.
```suggestion
        self.cpp_info.components["ut"].libs = []
```

Reply:
thats the default https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info

just remove the line please

```suggestion
```

Reply:
Ignore this, @prince-chrismc  is on the right track.

Reply:
I've already attempted to remove this line but it breaks the test package. Without that line, there is no `ut` component to link against, as the following error message demonstrates:

```sh
CMake Error at CMakeLists.txt:13 (add_executable):
  Target "test_package" links to target "boost::ut" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```

Reply:
which conan version are you using?

Reply:
```suggestion
        self.cpp_info.components["ut"].names["cmake_find_package"] = "ut"
        self.cpp_info.components["ut"].names["cmake_find_package_multi"] = "ut"
```

Reply:
Conan version 1.32.1

```sh
$ conan -v
Conan version 1.32.1
```

@madebr That change does work. Does that make it easier to understand?

Reply:
The point was to add some attribute to a `ut` component.
Setting only a `libs` attribute would have been fine too

Reply:
Ok, thanks. I'll mark this as resolved then.

Reply:
It's a bug. 🐛 

the defaults for cpp_info should be the packge_name which is the component name. but that's clearly not happening here 🤔 

/cc @danimtb is this the expected behavior?